### DISCUSSION TITLE
OneBot 12 标准进入候选状态！

### DISCUSSION THREAD

**Discussion by stdrc** [howmanybots/onebot#221] [通用]
在经过 [OSPP 2021](https://nonebot.dev/ospp-2021.html#%E8%AE%BE%E8%AE%A1-onebot-v12-%E6%8E%A5%E5%8F%A3%E6%A0%87%E5%87%86) 的最初设计和后续的 [一系列 RFC](https://github.com/botuniverse/onebot/issues?q=is%3Aissue+label%3AST-accepted) 后，OneBot 12 标准基本达到了稳定可行的状态。虽然社区中仍然有许多新颖的 idea 还没有进入标准，但我们认为目前是时候固定版本特性，发布候选版了。

OneBot 12 候选版发布后，将不再引入新的特性，而只是对文档的表述和排版进行错误修正。待社区开发者们做最后的跟进、检查后（预计两三周），将正式发布 OneBot 12。此后，新特性将进入 OneBot 13 草案。

**Comment by Asterism12** [comment]
现在是2022.12.29，请问OneBot正式发布的进展如何？

**Reply by stdrc** [reply]
目前标准内容已经基本稳定，也有了好几个实现，不过大家目前认为最好还是等有一点点生态之后再发布正式版。但标准内容在正式版之前肯定已经不会再做大改了，可以放心使用～

**Reply by soloctl** [reply]
感觉用词还是偏中式自己的理解, 不太准确.

**Reply by stdrc** [reply]
> 感觉用词还是偏中式自己的理解, 不太准确. 

不妨指出

**Reply by soloctl** [reply]
> > 感觉用词还是偏中式自己的理解, 不太准确.
> 
> 不妨指出

我就是嚣张一下, 没想到作者回复这么快? 客观来说, "不准确" 倒不一定, 毕竟发展到了v12, 全能自洽即可, 不用非得追求什么准确.  但有些和我经历的使用习惯不太一样, 我个人看起来有点出戏. 比如我列几点 :
1. Platform 一般是相对它自己, 它成为某种事的 platform , 比如相对于开发应用, 或者相对于连接bot , 但这些连接的平台很多它自身多不具备这种属性或者定位, 我感觉叫 channel 比较好,  对于channel的单个实例叫终端 terminal.
2. OneBot实现这个term本身没什么问题, 但使用时候也不用这么教条, 比如这句话 `通信方式规定了 OneBot 应用与 OneBot 实现进行网络连接所使用的通信协议、数据编码` ,  我感觉说`OneBot 应用端与 OneBot 标准 进行网络连接` 更自然一些.   OneBot 应用 本身并不一定是 OneBot 应用,  所以叫应用端或者标准的消费端,  会更恰当一些.  实现本身单独叫是正确的, 但在"相对"场合, 一般称自己为标准会更合适一点, 毕竟面对这些消费这个接口或实现的, 是接口或者说是标准, 它们并不在乎后边是哪个实现, 是不是完整或正确的实现. 
3. "HTTP Webhook" 叫 "HTTP Client" 或者 "HTTP Request" 辅以"层", "端"或"侧", 我个人感觉这样的叫法更自然一些.  

还没看全, 看到上边这几处个人第一反应略微有点出戏.
